ext/sqlite3/sqlite3.stub.php

```diff
@@ -57,22 +57,22 @@ public function exec(string $query) {}
     public function query(string $query) {}
 
     /** @return mixed */
-    public function querySingle(string $query, bool $entire_row = false) {}
+    public function querySingle(string $query, bool $entireRow = false) {}
 
     /** @return bool */
-    public function createFunction(string $name, callable $callback, int $argument_count = -1, int $flags = 0) {}
+    public function createFunction(string $name, callable $callback, int $argumentCount = -1, int $flags = 0) {}
```

Comment:
```suggestion
    public function createFunction(string $name, callable $callback, int $argCount = -1, int $flags = 0) {}
```
We abbreviate $argument everywhere